lib/manager/gradle/index.js

```diff
@@ -178,8 +174,24 @@ async function executeGradle(config) {
   return true;
 }
 
-function getDockerRenovateGradleCommandLine(localDir) {
-  return `docker run --rm -v ${localDir}:${localDir} -w ${localDir} renovate/gradle ${GRADLE_DEPENDENCY_REPORT_COMMAND}`;
+async function getGradleCommandLine(config) {
+  let cmd;
+  const gradlewExists = await fs.exists(config.localDir + '/gradlew');
+  if (gradlewExists && !config.gitFs) {
+    logger.warn(
+      'Found gradle wrapper (gradlew) in the project, but --git-fs is not enabled, not using it.'
+    );
+  }
+  if (config.gitFs && gradlewExists) {
+    cmd = 'sh gradlew';
+  } else if (config.binarySource === 'docker') {
+    cmd = `docker run --rm -v ${config.localDir}:${config.localDir} -w ${
+      config.localDir
+    } renovate/gradle gradle`;
```

Comment:
Probably the docker image should be configurable to allow users to define their own docker with specific versions and configurations

Reply:
Yes that would be great, but I believe this is out of the scope of this issue. Maybe we should create a separate issue for that?

Reply:
So if someone configures binarySource=docker then we ignore gradlew for now?

Reply:
For me that's what makes sense. Specially because the docker image could contain extra things that are required for running gradle for renovate. So, in some cases gradlew may not be suitable for renovate

Reply:
So does it seem OK to merge for now?